library/control/src/modules/ProductFeatures.rb

```diff
@@ -318,6 +318,25 @@ def Import(import_settings)
       nil
     end
 
+    # Overlay, or override, some features.
+    # The intended use is to use PushOverlay to apply some features
+    # specified by a system role (FATE#317481).
+    # PopOverlay would be called only when coming Back
+    # to the role selection dialog (and then PushOverlay with a different one)
+    # @param features [Hash{String => Hash{String => Object}}] in the same
+    #   format as `@features` or `@defaults`
+    def PushOverlay(features)
```

Comment:
as I mention above `reset` and `set_overlay` do same stuff without need to implement any stack